### PR TITLE
Fixed some russian translation and encoding for files

### PR DESCRIPTION
**Pull request by Blueberryy**

### CODE REVIEW

**Comment by Blueberryy** [comment]
> This should also be fixed in reactivedrop_russian.txt or it'll be overwritten on the next sync.

Just fixed

**Comment by Blueberryy** [comment]
i dunno why it shows changes in italian files though

**Comment by Blueberryy** [comment]
> Did a rebase for you. Does that look correct?

Yeah much better

File: resource/reactivedrop_russian.txt [hunk 1]
```diff
@@ -4686,39 +4686,39 @@
...
+"[english]asw_ob_lanasbridge_1t"		"Get access to the bridge."
 "asw_ob_lanasbridge_1"		"Отряд, мы получаем автоматические отчёты о том, что оружейный комплекс МВС на Планете 137 активировал свою систему безопасности. Нет ответа от местного главного администратора, МВС посылает нас для расследования."
 "[english]asw_ob_lanasbridge_1"		"Squad, we\'re getting automated reports that an IAF weapon facility on Planet 137 has activated its security system. With no response from the local head administrator, the IAF are sending us to investigate"
 "asw_ob_lanasbridge_1b"		"За пределами зоны посадки, необходима крайняя осторожность при пересечении минных полей с детектором движения."
 "[english]asw_ob_lanasbridge_1b"		"Extreme caution will be required to cross a field of motion-detecting mines beyond the landing area."
 "asw_ob_lanasbridge_1c"		"После этого вам нужно будет найти способ пробраться к мосту."
 "[english]asw_ob_lanasbridge_1c"		"After that, you\'ll have to find a way to get access to the bridge."
 "asw_ob_lanasbridge_2t"		"Перейдите мост."
-"[english]asw_ob_lanasbridge_2t"		"Cross the bridge"
+"[english]asw_ob_lanasbridge_2t"		"Cross the bridge."
```
Comment (BenLubar) [inline resource/reactivedrop_russian.txt:4697]: Changing the English string will just reset the translation to untranslated when I sync.

Comment (Blueberryy) [reply resource/reactivedrop_russian.txt:4697]: will change back to original asap

Comment (Blueberryy) [reply resource/reactivedrop_russian.txt]: changed

Comment (Blueberryy) [reply resource/reactivedrop_russian.txt]: should be all good now (100%)

**Pull request merged**